### ISSUE TITLE
Purchase + install coving

### ISSUE THREAD

**Issue by sarahmonster** [tofumatt/the-flat-that-matt-and-sarah-rebuilt#66]
Here are my notes for this!

skirting: 110+194+120+12+160
cove: 194+120+111+383+221+52+7+254+204

194 = 1
120 + 111 = 1x2.6
383 = 2
254 = 1x2.6
204 + 52 = 1x2.6
221 + 7 = 1x2.6

2.6 = 4
2 = 3

wm boyle 10% off welcome10
2-3 days delivery (order friday?)
free delivery over £90

16m of coving (8 lengths) (CX106 glasgow)
6m of skirting (4 lengths for wasteage!) (SX173 regency)
maybe also buy a length for the bookcase?

(measure to be sure!)
1
so we'd need deco fix hydro, covers 7-8m
3 or 4 cartridges
plus a cartridge of extra-fix fx200 for the joints

**Comment by sarahmonster** [comment]
So basically I purchased the following:

<img width="985" alt="screenshot 2018-08-31 14 35 12" src="https://user-images.githubusercontent.com/376315/44922128-66d9e880-ad3c-11e8-8290-0ec47562f0ba.png">

The dentil block coving is for the bookcase, and everything else is for the kitchen. I got a bit of extra length (ie we need 16m of coving, and I bought 18) so that we can muck it up as needed, make test cuts, and allow for wasteage. Also hopefully the ceiling isn't substantially taller than the floor. 

Once the plaster is dried and before we paint (#67), we'll go ahead and install the coving. The skirting will wait until we've done the tiling, so it'll be a while yet!
